src/OpenLoco/src/Map/Tile.h

```diff
@@ -29,6 +30,14 @@ namespace OpenLoco::World
     // 0x004F9296, 0x4F9298
     constexpr Pos2 offsets[4] = { { 0, 0 }, { 0, 32 }, { 32, 32 }, { 32, 0 } };
 
+    struct LessThanPos3
```

Comment:
I don't like having this in `Tile.h`... Suggestions for where to put this? It's used by `TileManager` and `TileClearance` as well as the `raiseland`, `lowerLand`, `clearLand` game commands.